### ISSUE TITLE
implicit declaration of function 'my_make_scrambled_password'

### ISSUE THREAD

**Issue by kurbar** [NigelCunningham/pam-MySQL#22]
After configuring the library and trying to make it, from the start it fails with the following:

```
pam_mysql.c: In function ‘pam_mysql_check_passwd’:
pam_mysql.c:2896:15: warning: implicit declaration of function ‘my_make_scrambled_password’ [-Wimplicit-function-declaration]
               my_make_scrambled_password(buf, passwd, strlen(passwd));
               ^
mv -f .deps/pam_mysql_la-pam_mysql.Tpo .deps/pam_mysql_la-pam_mysql.Plo
/bin/bash ./libtool  --tag=CC   --mode=link gcc  -g -O2 -I/usr/include/mysql -fabi-version=2 -fno-omit-frame-pointer   -module -avoid-version  -o pam_mysql.la -rpath /lib/security pam_mysql_la-pam_mysql.lo -lcrypto -lpam -lcrypt  -L/usr/lib/x86_64-linux-gnu -lmysqlclient -lpthread -lz -lm -lrt -ldl
libtool: link: gcc -shared  -fPIC -DPIC  .libs/pam_mysql_la-pam_mysql.o   -lcrypto -lpam -lcrypt -L/usr/lib/x86_64-linux-gnu -lmysqlclient -lpthread -lz -lm -lrt -ldl  -g -O2   -Wl,-soname -Wl,pam_mysql.so -o .libs/pam_mysql.so
libtool: link: ( cd ".libs" && rm -f "pam_mysql.la" && ln -s "../pam_mysql.la" "pam_mysql.la" )
```

Running Ubuntu 16.04 with MySQL 5.7

**Comment by wferi** [comment]
This isn't a failure but a warning. See issue #16 for a little more detail.

**Comment by NigelCunningham** [comment]
Should be fixed in 0.8.x branch. Please test if you're able.

**Comment by NigelCunningham** [comment]
Closing in preparation for a 0.8.1 release (I believe this is fixed now).
